docs/addons/ack-addon.md

```diff
@@ -0,0 +1,115 @@
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.AckAddOn({
+    skipVersionValidation: true
+}),
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+> Pattern # 2 : This installs AWS Controller for Kubernetes for RDS ACK controller. After Installing this RDS ACK Controller, the instructions in [Provision ACK Resource](https://preview--eksworkshop-v2-next.netlify.app/docs/gitops/controlplanes/ack/configureResources) can be used to provision Amazon RDS database using the RDS ACK controller as an example.
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.AckAddOn({
+  skipVersionValidation: true,
+  name: "rds-chart",
+  chart: "rds-chart",
+  version: "v0.1.1",
+  release: "rds-chart",
+  repository: "oci://public.ecr.aws/aws-controllers-k8s/rds-chart",
+  managedPolicyName: "AmazonRDSFullAccess",
+  createNamespace: false
+}),
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+> Pattern # 3 : This installs AWS Controller for Kubernetes for EC2 ACK controller. After Installing this RDS ACK Controller, the instructions in [Provision ACK Resource](https://preview--eksworkshop-v2-next.netlify.app/docs/gitops/controlplanes/ack/configureResources) can be used to provision EC2 namespaces `SecurityGroup` resources required for creating Amazon RDS database as an example.
+
+```typescript
+import * as cdk from 'aws-cdk-lib';
+import * as blueprints from '@aws-quickstart/eks-blueprints';
+
+const app = new cdk.App();
+
+const addOn = new blueprints.addons.AckAddOn({
+  skipVersionValidation: true,
+  name: "ec2-chart",
+  chart: "ec2-chart",
+  version: "v0.1.0",
+  release: "ec2-chart",
+  repository: "oci://public.ecr.aws/aws-controllers-k8s/ec2-chart",
+  managedPolicyName: "AmazonEC2FullAccess",
+  createNamespace: false
+}),
+
+const blueprint = blueprints.EksBlueprint.builder()
+  .addOns(addOn)
+  .build(app, 'my-stack-name');
+```
+
+## Configuration Options
+
+- `name`: Name of the ACK Chart
+- `chart`: Chart Name of the ACK Chart
+- `version`: Version of the ACK Chart
+- `release`: Release Name of the ACK Chart
+- `repository`: Repository URI of the specific ACK Chart
+- `managedPolicyName`: Policy Name required to be added to the IAM role for that ACK
+- `createNamespace`: (boolean) This should be false if you are using for the second time
+- `values`: Arbitrary values to pass to the chart. Refer to the cert-manager [Helm Chart documentation](https://artifacthub.io/packages/helm/cert-manager/cert-manager) for additional details. It also supports all standard helm configuration options ( for Eg: https://github.com/aws-quickstart/cdk-eks-blueprints/blob/main/docs/addons/index.md#standard-helm-add-on-configuration-options)

```

Comment:
Please drop the reference to the cert-manager here.

Reply:
Done.